### PR TITLE
Updates to legal page

### LINKED ISSUE #3115: /legal check copydoc for changes
Check copydoc for changes and implement accordingly. Copy doc: https://docs.google.com/document/u/1/d/19yTLYWu7rEErPrO0xDsqMeN8tcWKBjHuarw0tjxRlhY/edit?usp=drive_web&ouid=101513226872617209827

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Updates to `/legal` page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/legal](http://0.0.0.0:8001/legal)
- Check that page matches [the copy doc](https://docs.google.com/document/d/19yTLYWu7rEErPrO0xDsqMeN8tcWKBjHuarw0tjxRlhY/edit)

## Issue / Card
Fixes #3115

### CODE REVIEW

**Comment by Caleb-Ellis** [comment]
@steverydz @pmahnke Update from Mark in the copydoc - title should be "Legal Terms and Conditions" instead of "Ubuntu legal"

File: templates/legal/index.html [hunk 1]
```diff
@@ -23,76 +23,57 @@ <h1>Ubuntu legal</h1>
...
+        <h3>
+          <a href="/legal/ubuntu-advantage">Ubuntu Advantage terms&nbsp;&rsaquo;</a>
+        </h3>
+        <p>A legal description of the services included in the Ubuntu Advantage support package.</p>
       </div>
+      <div class="col-6 p-card">
+        <h3>
+          <a href="/legal/managed-services">Managed Services terms&nbsp;&rsaquo;</a>
+        </h3>
+        <p>A legal description of the services included in Bootstack and Managed Kubernetes.</p>
```
Comment (Caleb-Ellis) [inline templates/legal/index.html]: This is different to the copydoc
`A legal description of the services included in Bootstack.`

File: templates/legal/index.html [hunk 2]
```diff
@@ -23,76 +23,57 @@ <h1>Ubuntu legal</h1>
...
+  </div>
+  <div class="row">
+    <div class="col-12 p-card">
+      <h3>
+        <a href="/legal/service-terms">Service terms&nbsp;&rsaquo;</a>
+      </h3>
+      <p>This agreement sets out Canonical&rsquo;s standard service terms for both Ubuntu Advantage and Managed Services customers.</p>
     </div>
   </div>
 </div>
```
Comment (Caleb-Ellis) [inline templates/legal/index.html]: The copydoc shows a four column footer after this?

Comment (steverydz) [reply templates/legal/index.html]: @pmahnke Does that ^ seem right to you? We don't have a 4 col footer anywhere else as far as I can see.

File: templates/legal/index.html (hunk 1 above)
Comment (steverydz) [reply templates/legal/index.html]: Thanks, fixed

**Comment by steverydz** [comment]
@pmahnke This is the issue we mentioned in stand up with the 4 col footer

**Comment by frankiegranato** [comment]
@steverydz I think the 4 col footer issue is no longer in the copydoc (It wasn't a Mark edit it was just out of date). Also, please update the heading and page description (as per copydoc)

**Comment by steverydz** [comment]
@Caleb-Ellis @frankiegranato Both done

**Pull request merged**